Chrome_Extenstion/FrontEndFiles/Javascript/newtab.js

```diff
@@ -0,0 +1,17 @@
+let mainBody = document.getElementById("main_body");
+
+var item = localStorage.getItem('name');
```

Comment:
never worked with localStorage... seems quite simple. awsome

Reply:
I've never worked with local storage either, for this did you need to include in permission in manifest as well?